tests/Jellyfin.Server.Implementations.Tests/BaseItem/BaseItemKindTests.cs

```diff
@@ -0,0 +1,97 @@
+
+            Assert.NotNull(defaultConstructor);
+            if (defaultConstructor != null)
+            {
+                var instance = (MediaBrowser.Controller.Entities.BaseItem)defaultConstructor.Invoke(null);
+                var exception = Record.Exception(() => instance.GetBaseItemKind());
+                Assert.Null(exception);
+            }
+        }
+
+        private class GetBaseItemDescendant : IEnumerable<object?[]>
+        {
+            private static bool IsProjectAssemblyName(string? name)
+            {
+                if (name == null)
+                {
+                    return false;
+                }
+
+                return name.StartsWith("Jellyfin", StringComparison.OrdinalIgnoreCase)
+                       || name.StartsWith("Emby", StringComparison.OrdinalIgnoreCase)
+                       || name.StartsWith("MediaBrowser", StringComparison.OrdinalIgnoreCase)
+                       || name.StartsWith("RSSDP", StringComparison.OrdinalIgnoreCase);
+            }
+
+            public IEnumerator<object?[]> GetEnumerator()
+            {
+                var loadedAssemblies = AppDomain.CurrentDomain.GetAssemblies();
+                foreach (var assembly in loadedAssemblies)
+                {
+                    if (IsProjectAssemblyName(assembly.FullName))
+                    {
+                        var baseItemTypes = assembly.GetTypes()
+                            .Where(targetType => targetType.IsClass
+                                                 && !targetType.IsAbstract
+                                                 && targetType.IsSubclassOf(typeof(MediaBrowser.Controller.Entities.BaseItem)));
+                        foreach (var baseItemType in baseItemTypes)
+                        {
+                            yield return new object?[] { baseItemType };
+                        }
+                    }
+                }
+
+                var path = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                if (path == null)
+                {
+                    throw new NullException("Assembly location is null");
+                }
+
+                foreach (string dll in Directory.GetFiles(path, "*.dll"))
+                {
+                    var assembly = Assembly.LoadFile(dll);
+                    if (IsProjectAssemblyName(assembly.FullName))
+                    {
+                        var baseItemTypes = assembly.GetTypes()
+                            .Where(targetType => targetType.IsClass
+                                                 && !targetType.IsAbstract
+                                                 && targetType.IsSubclassOf(typeof(MediaBrowser.Controller.Entities.BaseItem)));
+                        foreach (var baseItemType in baseItemTypes)
+                        {
+                            yield return new object?[] { baseItemType };
+                        }
+                    }
+                }
```

Comment:
Not sure why this is needed?

Reply:
Somehow not all assemblies are loaded in so this test was missing a class that inherits from `BaseItem`